addons/redis/config/redis7-config.tpl

```diff
@@ -43,9 +43,10 @@ appendfsync everysec
 no-appendfsync-on-rewrite no
 auto-aof-rewrite-percentage 100
 auto-aof-rewrite-min-size 64mb
+aof-disable-auto-gc no
 aof-load-truncated yes
 aof-use-rdb-preamble yes
-aof-timestamp-enabled no
+aof-timestamp-enabled yes
```

Comment:
If upgrade, will these parameter change affect the existing cluster?

Reply:
Only `aof-timestamp-enabled yes` will take effect. In the later aof log, each command will be attached with a timestamp annotation, is transparent to the user.